weldx/schemas/weldx.bam.de/weldx/core/media_file-0.1.0.yaml

```diff
@@ -0,0 +1,38 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "asdf://weldx.bam.de/weldx/schemas/core/media_file-0.1.0"
+
+title: |
+  Schema for a media file (time-based and single shot).
+description: |
+  This schema describes a media file by compiling all its meta data.
+  Optionally, the whole file can also be stored in the
+  binary block of an asdf file.
+
+examples:
+  -
+    - foo
+    - bar
+
+type: object
+properties:
+  file:
+    description: |
+      The basic file information/content.
+    tag: "asdf://weldx.bam.de/weldx/tags/core/file-0.1.1"
+  recorded_at:
+    description: |
+      Time when this media was recorded.
+    tag: "asdf://weldx.bam.de/weldx/tags/time/timestamp-0.1.*"
+  fps:
+    description: |
+      In case of time-dynamic media, the time resolution (frames per second).
+    tag: "asdf://weldx.bam.de/weldx/tags/units/quantity-0.1.0"
+
+propertyOrder: [file, recorded_at, fps]
+anyOf:
+  - required: [file, recorded_at]
```

Comment:
are you planning on adding different variations here?
Otherwise the `anyOf` block is not needed

Reply:
So far, I do not. Will remove it then.